pwndbg/commands/one_gadget.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import argparse
+import shutil
+
+import pwndbg.color.message as M
+import pwndbg.commands
+import pwndbg.gdblib.one_gadget
+import pwndbg.glibc
+from pwndbg.commands import CommandCategory
+
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.RawTextHelpFormatter,
+    description="""Show one_gadget
+
+Examples:
+    one_gadget
+    one_gadget --show-unsat
```

Comment:
The command has been renamed to `onegadget`

Reply:
Ahh, yes, thanks, I'll fix it!